### PR TITLE
refactor: fix processing bootstrap, several tests, change test_db setup to alembic migrations

### PR DESCRIPTION
**Pull request by spiridonovfed**
…p to alembic migrations

### CODE REVIEW

File: processing/Dockerfile [hunk 1]
```diff
@@ -14,23 +14,15 @@ ENV PYTHONUNBUFFERED 1
 FROM base as build
 COPY alembic alembic
 COPY src src
-CMD alembic upgrade head && uvicorn src.main:app --host 0.0.0.0 --port 8080
+CMD alembic upgrade head && uvicorn src.main:app --host 0.0.0.0 --port 8080 --reload
```
Comment (cakeinsauce) [inline processing/Dockerfile:17]: May you clarify usage of `reload` here?

File: processing/tests/conftest.py [hunk 2]
```diff
@@ -67,24 +74,43 @@ def files_data_for_pipeline(files_data):
...
-    Base.metadata.create_all(bind=engine)
-    session_local = sessionmaker(
-        autocommit=False, autoflush=False, bind=engine
-    )
-    session = session_local()
-    yield session
-    session.close()
-    Base.metadata.drop_all(bind=engine)
-    remove_db(test_db_name)
+def test_db_engine():
```
Comment (cakeinsauce) [inline processing/tests/conftest.py:87]: May you rename it to `testing_db_engine` or `db_test_engine` so we won't confuse it with tests

File: processing/tests/conftest.py [hunk 3]
```diff
@@ -67,24 +74,43 @@ def files_data_for_pipeline(files_data):
...
+    except SQLAlchemyError as e:
+        raise SQLAlchemyError(f"Got an Exception during migrations - {e}")
+
+    yield
+
+    drop_database(test_db_engine.url)
+
+
+@pytest.fixture
+def test_db_session(test_db_engine, setup_database):
```
Comment (cakeinsauce) [inline processing/tests/conftest.py:110]: May you rename it the same way as testing engine?

File: processing/Dockerfile (hunk 1 above)
Comment (spiridonovfed) [reply processing/Dockerfile:17]: I used it during development so the webapp could reload after changes in code.
I've deleted it here

File: processing/tests/conftest.py (hunk 2 above)
Comment (spiridonovfed) [reply processing/tests/conftest.py:87]: agreed - renamed to db_test_engine

File: processing/tests/conftest.py (hunk 3 above)
Comment (spiridonovfed) [reply processing/tests/conftest.py:110]: agreed - renamed to db_test_session

**Review by cakeinsauce — APPROVED** [review]

**Pull request merged**